.github/workflows/test-contracts.yml

```
@@ -28,21 +28,16 @@ jobs:
-  #       run: npm run coverage
-  #     - name: Upload Code Coverage
-  #       uses: codecov/codecov-action@v5
-  #       with:
-  #         token: ${{ secrets.CODECOV_TOKEN }}
-  #         fail_ci_if_error: true
+  unit-test:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    if: github.actor != 'secured-finance-machine-user[bot]'
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)